PDBminer

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env python3
 
-# Copyright (C) 2022, Kristine Degn
+# Copyright (C) 2021-2023, Kristine Degn
```

Comment:
"The general rule is that the year to include in a copyright notice is the year of first publication of the work. First publication is when the work is made available to the public without restriction."

so I think it should be just 2023 :)